src/js/lotto.js

```diff
@@ -1,35 +1,48 @@
-import { getRandomNumbers, shuffle } from "./utils.js";
+import { getRandomNumbers, shuffle } from './utils.js';
+import { $lottoIcons } from './dom.js';
 
-const $lottoIcons = document.querySelector(".lotto-icons");
-export class Lotto {
-  static MAX = 45;
-  static SIZE = 6;
-}
-
-export class LottoTicket {
+export class LottoModel {
+  #MAX = 45;
+  #SIZE = 6;
```

Comment:
```suggestion
export class LottoModel {
  static #MAX = 45;
  static #SIZE = 6;
```
이렇게 해야 상수로 사용할 수 있을 것 같아요!